packages/block-library/src/heading/deprecated.js

```diff
@@ -48,7 +52,52 @@ const migrateCustomColors = ( attributes ) => {
 	};
 };
 
+const migrateTextAlign = ( attributes ) => {
+	const { align, ...rest } = attributes;
+	return TEXT_ALIGN_OPTIONS.includes( align )
+		? { ...rest, textAlign: align }
+		: attributes;
+};
+
+const TEXT_ALIGN_OPTIONS = [ 'left', 'right', 'center' ];
+
 const deprecated = [
+	{
+		supports: {
+			className: false,
+			anchor: true,
+			align: [ 'wide', 'full' ],
+			color: { link: true },
+			fontSize: true,
+			lineHeight: true,
+			__experimentalSelector: {
+				'core/heading/h1': 'h1',
+				'core/heading/h2': 'h2',
+				'core/heading/h3': 'h3',
+				'core/heading/h4': 'h4',
+				'core/heading/h5': 'h5',
+				'core/heading/h6': 'h6',
+			},
+			__unstablePasteTextInline: true,
+		},
+		attributes: { ...blockAttributes },
```

Comment:
What's the reason for using shallow copy here?

Reply:
No reason. Left it there early on my first changes. I'll change it, thanks!